extractor/src/main.py

```diff
@@ -126,10 +138,9 @@ def handle_authorization_error(e):
 # KAFKA CLIENT #
 ################
 
+CONSUMED_TOPICS = "^batch\\..*"
```

Comment:
Why are there 2 `\` here?

Reply:
Because the topic naming format is "{type}.{uuid}". I'm including the dot in the regex.